Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -377,7 +378,7 @@ require({
                     errorLines.push(line);
                 }
             }
-        }
+        }/*eslint-enable new-cap*/
```

Comment:
We probably want to establish a style guide here, I propose all `eslint-` statements in the code be on their own line.